src/fragments/lib/datastore/react-native/getting-started/30_platformIntegration.mdx

```diff
@@ -28,10 +22,9 @@ React Native CLI apps can now use SQLite with DataStore. SQLite offers considera
 To enable SQLite with DataStore for enhanced local database performance, follow the steps below:
 
 ```sh
```

Comment:
Will be there be difference on the rendered results using `sh` vs `bash` above?

Reply:
I'll change the ones I see to `sh` for consistency